contracts/core/src/contract.rs

```diff
@@ -174,7 +174,7 @@ fn query_exchange_rate(deps: Deps<NeutronQuery>, config: &Config) -> ContractRes
         )?;
 
     let delegations_amount: Uint128 = delegations
```

Comment:
do you mind renaming `delegations` var into `delegations_response` or smth as `delegations.delegations.delegations` looks pretty weird